src/main/scala/org/broadinstitute/dsde/snoop/ws/SnoopApiFormats.scala

```diff
@@ -5,6 +5,17 @@ import spray.json._
 import com.wordnik.swagger.annotations.{ApiModel, ApiModelProperty}
 import scala.annotation.meta.field
 
+/**
+ * Marker trait for normal messages
+ */
+trait RestMessage
+
+/**
+ * Message used to signify that a validation error happened
+ * @param message
+ */
+case class Validation(message: String)
```

Comment:
I am happy to call it ValidationMessage, it is not even used. 

I am not sure I understand your second comment, is it that you like these base traits or not?